components/prism-bqn.js

```diff
@@ -0,0 +1,61 @@
+Prism.languages.bqn = {
+	'shebang': {
+	pattern: /^#!\s*\/.*/,
+	alias: 'important'
+	},
+	'comment': /#.*$/m,
+	'string-literal': {
+	pattern: /"(?:\\[\s\S]|\$(?!\()|`[^`]+`|[^"\\`$])*"/,
+	greedy: true,
+	alias: 'string'
+	},
+	'character-literal': {
+	pattern: /'.'/,
+	greedy: true,
+	alias: 'string'
+	},
+	'function': {
+	pattern: /[•][A-Z0-9_¯.∞π]+[A-Z0-9_¯.∞π]*/i
+	},
+	'dot-notation-on-brackets': {
+		pattern: /{(?=.*}\.)|}\./,
+		alias: 'namespace'
+	},
+	'special-name': {
+	pattern: /[𝕨𝕩𝕗𝕘𝕤𝕣𝕎𝕏𝔽𝔾𝕊]|_𝕣[_]?/,
+	alias: 'keyword'
+	},
+	'dot-notation-on-name': {
+		pattern: /[A-Z_][A-Z_¯∞π0-9]*\./i,
+		alias: 'namespace'
+	},
+	'word-number-scientific': {
+		pattern: /\d+(?:\.\d+)?[e|E][¯]?\d+/,
+		alias: 'number'
+	},
+	'word-name': {
+	pattern: /[A-Z_][A-Z_¯∞π0-9]*/i,
+	alias: 'symbol'
+	},
+	'word-number': {
+		pattern: /[¯∞π]?(?:\d*\.?\b\d+(?:e[+¯]?\d+)?|¯|∞|π)(?:j¯?(?:(?:\d+(?:\.\d+)?|\.\d+)(?:e[+¯]?\d+)?|¯|∞|π))?/i,
+		alias: 'number'
+	},
+	'null-literal': {
+	pattern: /[@]/,
+	alias: 'string'
+	},
+	'primitive-functions': {
+	pattern: /[-+×÷⋆√⌊⌈|¬∧∨<>≠=≤≥≡≢⊣⊢⥊∾≍⋈↑↓↕«»⌽⍉/⍋⍒⊏⊑⊐⊒∊⍷⊔!]/u,
```

Comment:
We still support browsers that don't support the u flag. Luckily, all these characters are one UTF16 char code, so we can safely remove the u flag without affecting the pattern.
```suggestion
	pattern: /[-+×÷⋆√⌊⌈|¬∧∨<>≠=≤≥≡≢⊣⊢⥊∾≍⋈↑↓↕«»⌽⍉/⍋⍒⊏⊑⊐⊒∊⍷⊔!]/,
```

Reply:
Unfortunately, it seems that there are surrogate pairs within the `special-name` tokens (`𝕨𝕩𝕗𝕘𝕤𝕣𝕎𝕏𝔽𝔾𝕊 _𝕣 _𝕣_`). If I should avoid the `/u` modifier, what is the way to deal with these?

Thank You for the prompt feedback!

Reply:
Those have to be made into a simple groups, unfortunately. So `(?:𝕨|𝕩|𝕗|𝕘|𝕤|...)`.

Reply:
Ahh, Okay. Done and done.